### ISSUE TITLE
Paint gun activation field error display

### ISSUE THREAD

**Issue by vrad-exe** [BEEmod/BEE2-items#4411]
Using paint gun activation fields in games besides Aperture Tag should produce an error display, instead of silently swapping it to a regular fizzler, as the current behavior has resulted in people thinking the item is broken.

(This is kind of a stopgap until #4296 is implemented)

**Comment by LincoCompany00** [comment]
Can the paint gun activation field be configured in so if you use it in base Portal 2, it will generate as a paint gun activation field and have the same effects to the portal gun as it does in Aperture Tag? That would allow for players to make maps that are supposed to be paint gun maps, except the player has to solve it with the portal gun!

**Comment by vrad-exe** [comment]
Oh like portal gun activation fields? That's an interesting idea, though probably should be a separate item considering we do plan to eventually make the paint gun available in base Portal 2.

**Comment by LincoCompany00** [comment]
Yeah, the Portal gun activation field should be a separate item; I've actually suggested it before, actually. I think, in a different issue.
Here, I meant that if you use the paint gun activation field in Portal 2, it would become a Paint Gun Activation Field, like in Aperture Tag. But it would affect the Portal gun the same way as it affects P Body/Altas' portal gun in Co-op; for if the test chamber was originally designed as a paint gun test, but you have to solve it with the portal gun!

**Comment by vrad-exe** [comment]
Not sure how that would work, the portal gun and paint gun aren't really interchangeable, you'd probably just end up stuck if you tried to complete a map designed for the paint gun with the portal gun or vice versa

**Comment by LincoCompany00** [comment]
I mean that... the maker of the map would create a map, and the story behind it would be that the test chamber, in the world of Portal 2, was designed for the paint gun. But in reality, creator of the map designed it specifically so it can be solved with either!
I guess it's kind of like in Aperture Tag, where Nigel keeps having to recycle older test chambers— and those test chambers are the same ones from the Portal 2 campaign. But you'd solve it with the paint gun, because you have it instead of a portal gun. It would work like that, but the other way around! But the creator of the BEE2 map would design it so it can be solved with either device.
That's really just one way someone could use it. I don't really know how to explain it very well

**Comment by vrad-exe** [comment]
Storytelling like that doesn't work very well in puzzlemaker maps, and either way you could do the same thing with a separate portal gun activation field item? (Or just giving the player a portal gun without an activation field at all)
